diff --git a/head_src/guihead/guihead.c b/head_src/guihead/guihead.c
index f0d9935..c6599f0 100644
--- a/head_src/guihead/guihead.c
+++ b/head_src/guihead/guihead.c
@@ -120,7 +120,7 @@ int APIENTRY WinMain(HINSTANCE hInstance,
 	{
 		if (splash || stayAlive)
 		{
-			if (!SetTimer (hWnd, ID_TIMER, 1000 /* 1s */, TimerProc))
+			if (!SetTimer (hWnd, ID_TIMER, 100, TimerProc))
 			{
 				signalError();
 				return 1;
